server/src/main/java/keywhiz/auth/cookie/AuthenticatedEncryptedCookieFactory.java

```diff
@@ -118,15 +118,7 @@ public NewCookie getSessionCookie(User user, ZonedDateTime expiration) {
    * @return serialized expired cookie with matching parameters to authenticating cookie.
    */
   public NewCookie getExpiredSessionCookie() {
-    HttpCookie cookie = new HttpCookie(config.getName(), "expired", config.getDomain(), config.getPath(),
-        0, config.isHttpOnly(), config.isSecure());
-
-    Response response = newResponse();
-    response.addCookie(cookie);
-    return NewCookie.valueOf(response.getHttpFields().get(HttpHeader.SET_COOKIE));
-  }
-
-  private Response newResponse() {
-    return new Response(new HttpChannel(null, new HttpConfiguration(), null, null), null);
```

Comment:
Creating an HttpChannel with a null Connector is not supported after the Jetty upgrade in Dropwizard, but it appears that the only purpose of this Response was to convert HttpCookie to NewCookie.